lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -181,7 +186,12 @@ private function importRow($sourceActorId, $targetActorId, $relationTypeId)
         foreach (array_unique(array_merge($relations, $relationsAlternate)) as $relationId)
         {
           $relation = QubitRelation::getById($relationId);
```

Comment:
I think this should be moved into the conditional below (with `$relation->delete()`), to avoid the cost of querying the database unnecessarily.

Reply:
Good call!